answer_rocket/config.py

```diff
@@ -216,6 +216,95 @@ def delete_copilot_skill_chat_question(self, copilot_skill_chat_question_id: UUI
         except Exception as e:
             return None
 
+    def create_copilot_question(self, nl='') -> MaxMutationResponse:
+        try:
+            mutation_args = {
+                'copilotId': self.copilot_id,
+                'copilotQuestion': {
+                    'nl': nl
+                }
+            }
+
+            mutation_vars = {
+                'copilot_id': Arg(non_null(GQL_UUID)),
+                'copilot_question': Arg(non_null(MaxCopilotQuestionInput))
+            }
+
+            operation = self._gql_client.mutation(variables=mutation_vars)
+
+            operation.create_max_copilot_question(
+                copilot_id=Variable('copilot_id'),
+                copilot_question=Variable('copilot_question')
+            )
+
+            result = self._gql_client.submit(operation, mutation_args)
+
+            max_mutation_response = result.create_max_copilot_question
+
+            return max_mutation_response
+        except Exception as e:
+            return None
+
+    def update_copilot_question(self, copilot_question_id: str, nl='') -> MaxMutationResponse:
+        try:
+            mutation_args = {
+                'copilotId': self.copilot_id,
+                'copilotQuestionId': copilot_question_id,
```

Comment:
Inconsistent with `delete_copilot_chat_question`, which is a UUID.  Probably makes sense to have the `copilot_question_id` arg as a UUID, and massage as needed for GraphQL.